### PR TITLE
#113 fix js hint warning

### PR DESCRIPTION
**Pull request by Andriets**
#113 It needs to be finalized in the future

### CODE REVIEW

File: EventsExpress/ClientApp/src/components/profile/editProfile/change-avatar.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ class ChangeAvatar extends React.Component {
     handleFile(fieldName, event) {
         event.preventDefault();
         // convert files to an array
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/components/profile/editProfile/change-avatar.js]: Seems like this piece of code does not do anything. Remove it all please.

File: EventsExpress/ClientApp/src/components/user-info/user-block.js [hunk 2]
```diff
@@ -7,11 +7,9 @@ import IconButton from "@material-ui/core/IconButton";
 export function UserBlock(props) {
     const { user, isCurrentUser, block, unblock} = props;
 
-    function onBlock(){props.openDialog({title: "Block user",message: "Are u sure?",action: props.block})}
-
     return (isCurrentUser) ? <td> </td> : <> <td className="align-middle">
                 <div className="d-flex justify-content-center align-items-center">
-                {(user.isBlocked == true)
+                {(user.isBlocked === true)
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/components/user-info/user-block.js]: No need to compare bool value to true. Just use the value itself.

File: EventsExpress/EventsExpress.csproj [hunk 3]
```diff
@@ -91,4 +91,6 @@
     </ItemGroup>
   </Target>
 
+  <ProjectExtensions><VisualStudio><UserProperties clientapp_4_1jshintrc__JsonSchema="" /></VisualStudio></ProjectExtensions>
```
Comment (iuriikhrystiuk) [inline EventsExpress/EventsExpress.csproj]: Why is this piece required? Should we remove it?

**Review by iuriikhrystiuk — CHANGES_REQUESTED** [review]

File: EventsExpress/EventsExpress.csproj (hunk 3 above)
Comment (Andriets) [reply EventsExpress/EventsExpress.csproj]: This line was deleted

File: EventsExpress/ClientApp/src/components/comment/comment-item.js [hunk 4]
```diff
@@ -1,36 +1,31 @@
...
+        if (days !== 0) return `${days} days ago`;
         time -= days * 24 * 60 * 60 * 1000;
         var hours = Math.floor(time / 60 / 60 / 1000);
-        if (hours != 0) return `${hours} hours ago`;
+        if (hours !== 0) return `${hours} hours ago`;
         time -= hours * 60 * 60 * 1000;
         var minutes = Math.floor(time / 60 / 1000);
-        if (minutes != 0) return `${minutes} minutes ago`;
+        if (minutes !== 0) return `${minutes} minutes ago`;
         return `right now`;
```
Comment (sand0) [inline EventsExpress/ClientApp/src/components/comment/comment-item.js]: pls, replace this by library method (see [chat message](https://github.com/EventsExpress-SoftServe/EventsExpress/blob/b0fed1a65297dd666a1540c96bc20c672979faff/EventsExpress/ClientApp/src/components/chat/msg.js#L18) for example)

File: EventsExpress/ClientApp/src/components/user-info/user-block.js (hunk 2 above)
Comment (sand0) [reply EventsExpress/ClientApp/src/components/user-info/user-block.js]:
```
if (user.isBlocked.toString() === "True" ||
    user.isBlocked.toString() === "true" ||
    user.isBlocked.toString() === "TRUE" ||
) { just kidding }
```

**Review by iuriikhrystiuk — APPROVED** [review]

**Pull request merged**